doc/example/gradient_check.ipynb

```diff
@@ -0,0 +1,894 @@
+       "      <td id=\"T_d9255_row7_col0\" class=\"data row7 col0\" >0.000220</td>\n",
+       "      <td id=\"T_d9255_row7_col1\" class=\"data row7 col1\" >0.186953</td>\n",
+       "      <td id=\"T_d9255_row7_col2\" class=\"data row7 col2\" >-0.186522</td>\n",
+       "      <td id=\"T_d9255_row7_col3\" class=\"data row7 col3\" >0.000216</td>\n",
+       "      <td id=\"T_d9255_row7_col4\" class=\"data row7 col4\" >0.373475</td>\n",
+       "      <td id=\"T_d9255_row7_col5\" class=\"data row7 col5\" >0.000004</td>\n",
+       "      <td id=\"T_d9255_row7_col6\" class=\"data row7 col6\" >0.019104</td>\n",
+       "      <td id=\"T_d9255_row7_col7\" class=\"data row7 col7\" >0.000010</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th id=\"T_d9255_level0_row8\" class=\"row_heading level0 row8\" >sd_rSTAT5A_rel</th>\n",
+       "      <td id=\"T_d9255_row8_col0\" class=\"data row8 col0\" >-0.000259</td>\n",
+       "      <td id=\"T_d9255_row8_col1\" class=\"data row8 col1\" >18.588643</td>\n",
+       "      <td id=\"T_d9255_row8_col2\" class=\"data row8 col2\" >-18.589168</td>\n",
+       "      <td id=\"T_d9255_row8_col3\" class=\"data row8 col3\" >-0.000262</td>\n",
+       "      <td id=\"T_d9255_row8_col4\" class=\"data row8 col4\" >37.177811</td>\n",
+       "      <td id=\"T_d9255_row8_col5\" class=\"data row8 col5\" >0.000003</td>\n",
+       "      <td id=\"T_d9255_row8_col6\" class=\"data row8 col6\" >0.011911</td>\n",
+       "      <td id=\"T_d9255_row8_col7\" class=\"data row8 col7\" >0.000000</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n"
+      ],
+      "text/plain": [
+       "<pandas.io.formats.style.Styler at 0x7f760b67c810>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def highlight_value_above_threshold(x, threshold=10):\n",
+    "    return ['color: darkorange' if xi > threshold else None for xi in x]\n",
+    "\n",
+    "gc.style.apply(\n",
+    "    highlight_value_above_threshold, subset=[\"fd_err\"],\n",
+    ").background_gradient(\n",
+    "    cmap=sns.light_palette(\"purple\", as_cmap=True), subset=[\"abs_err\"],\n",
+    ").background_gradient(\n",
+    "    cmap=sns.light_palette(\"red\", as_cmap=True), subset=[\"rel_err\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to \"fix\" my gradients?\n",
+    "\n",
+    "- Find suitable simulation tolerances.\n",
+    "- Consider switching from adjoint to forward sensitivities, which tend to be more robust.\n",
+    "- Check the simulation logs for Warnings and Errors.\n",
+    "- Ensure that the model is correct.\n",
+    "- Be aware that, depending on the tolerances, the gradient might differ when it is calculated with sensitivities, because `Objective(x, sensi_orders=(0,))` provides a different value than `Objective(x, sensi_orders=(0,1))`"
```

Comment:
```suggestion
    "- Be aware that, depending on the tolerances, the gradient might differ when it is calculated with sensitivities, because `Objective(x, sensi_orders=(0,))` might provide a different value than `Objective(x, sensi_orders=(0,1))`"
```